audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/AuditApplication.java

```diff
@@ -23,15 +23,22 @@
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
 
-import java.util.Arrays;
-
+/**
+ * SpringBoot application entry-point class for audit-service executable
+ * All spring components to be scanned must be in this package
+ */
 @EnableEurekaClient
```

Comment:
Shouldn't this be @EnableDiscoveryClient?

Reply:
I'll pull develop